src/pudl/extract/xbrl.py

```diff
@@ -101,7 +107,7 @@ def convert_form(
     form: XbrlFormNumber,
     datastore: FercXbrlDatastore,
     output_path: Path,
-    clobber: bool,
+    sql_path: Path,
```

Comment:
i know the arg docs are currently a lil out of date but it'd be good to update em with this change.